### PR TITLE
Implement live coffeecups on the website.

### PR DESCRIPTION
**Pull request by guidobouman**

### CODE REVIEW

File: to-become-packages/coffeecounter/client/template.coffee [hunk 1]
```diff
@@ -1,2 +1,10 @@
 Template.numCupsOfCoffee.helpers
-  numCupsOfCoffee: -> CoffeeCounter.findOne()?.count or 0
+  numCupsOfCoffee: ->
+    d = new Date()
+    d.setUTCHours(0,0,0,0)
+    CoffeeCounter.find(
+      {
+        published_at: { $gte: d.toISOString() },
+        location: "q020"
```
Comment (Primigenus) [inline to-become-packages/coffeecounter/client/template.coffee]: If this only shows coffee consumed in Amsterdam, we should probably mention that on the page somewhere too. Any thoughts?

File: to-become-packages/coffeecounter/lib/shared.coffee [hunk 2]
```diff
@@ -1 +1,2 @@
-@CoffeeCounter = new Mongo.Collection("coffeecounter")
+iot_api = DDP.connect('https://iot-api.scalingo.io/');
```
Comment (Primigenus) [inline to-become-packages/coffeecounter/lib/shared.coffee]: Can you use camelcasing for the naming here for consistency?

File: to-become-packages/coffeecounter/client/template.coffee (hunk 1 above)
Comment (guidobouman) [inline to-become-packages/coffeecounter/client/template.coffee]: Completely right! This was just a quick PR to get this rolling again.

The coffee in The Hague isn't live yet. Should we artificially fill this, or clearly state it's from Amsterdam?

Comment (lukasvan3l) [inline to-become-packages/coffeecounter/client/template.coffee]: How about having a question/info icon next to it that links to a blog post that explains how we got the number an what tech was involved?

Comment (guidobouman) [inline to-become-packages/coffeecounter/client/template.coffee]: Ooh, good idea, but wouldn't it be cool if we had the The Hague numbers in as well? The sheer difference in approach is what makes it even more awesome!

Comment (lukasvan3l) [inline to-become-packages/coffeecounter/client/template.coffee]: Sure, if that's almost finished! Just don't wait too long on it :)

**Comment by tacoe** [comment]
Hey all, let's just change this to say Amsterdam in the copy and merge it in. Guido, I just talked to Ted, they are getting closer but it's going to take a while.

File: to-become-packages/coffeecounter/lib/shared.coffee [hunk 3]
```diff
@@ -1 +1,2 @@
-@CoffeeCounter = new Mongo.Collection("coffeecounter")
+IotApi = DDP.connect('https://iot-api.scalingo.io/');
```
Comment (sjoerdvisscher) [inline to-become-packages/coffeecounter/lib/shared.coffee]: Ah cool, dan kan ik de lift ook makkelijk toevoegen!

Comment (guidobouman) [inline to-become-packages/coffeecounter/lib/shared.coffee]: YES! Dat kan!

**Comment by guidobouman** [comment]
@Primigenus @tacoe I'm having merge conflicts, the coffeecounter is gone?

**Comment by guidobouman** [comment]
@lukasvan3l  :point_up:

**Pull request closed without merging**